### PR TITLE
Add composable testnet 

### PR DESCRIPTION
**Pull request by josietyleung**

### CODE REVIEW

File: osmo-test-5/osmosis.zone_assets.json [hunk 1]
```diff
@@ -54,6 +54,11 @@
       "chain_name": "chain4energytestnet",
       "base_denom": "uc4e",
       "path": "transfer/channel-111/uc4e"
+    },
+    {
+      "chain_name": "composabletestnet",
+      "base_denom": "ppica",
```
Comment (JeremyParish69) [inline osmo-test-5/osmosis.zone_assets.json:60]: I see here base denom 'ppica'. Just want to make sure 'ubanksy' is definitely not the base asset? Can we please correct the [assetlist.json](https://github.com/cosmos/chain-registry/blob/master/testnets/composabletestnet/assetlist.json) at the Chain Registry first?

Comment (josietyleung) [reply osmo-test-5/osmosis.zone_assets.json:60]: hi @JeremyParish69 yes the base denom is 'ppica' now, have updated it in the assetlist.json on chain registry. Thanks!

**Comment by JeremyParish69** [comment]
Ah, the check failure now is due to the ibc channel not having been registered either. I think this should be the last requirement.

**Comment by josietyleung** [comment]
> Ah, the check failure now is due to the ibc channel not having been registered either. I think this should be the last requirement.

@JeremyParish69 ah yes we opened a new channel, have updated it to reflect thanks!

**Comment by JeremyParish69** [comment]
> > Ah, the check failure now is due to the ibc channel not having been registered either. I think this should be the last requirement.
> 
> @JeremyParish69 ah yes we opened a new channel, have updated it to reflect thanks!

A new channel is fine since nothing has been registered. Can we please get this channel registered to the [Chain Registry](https://github.com/cosmos/chain-registry/tree/master/_IBC)?

**Comment by josietyleung** [comment]
> > > Ah, the check failure now is due to the ibc channel not having been registered either. I think this should be the last requirement.
> > 
> > 
> > @JeremyParish69 ah yes we opened a new channel, have updated it to reflect thanks!
> 
> A new channel is fine since nothing has been registered. Can we please get this channel registered to the [Chain Registry](https://github.com/cosmos/chain-registry/tree/master/_IBC)?



> > > Ah, the check failure now is due to the ibc channel not having been registered either. I think this should be the last requirement.
> > 
> > 
> > @JeremyParish69 ah yes we opened a new channel, have updated it to reflect thanks!
> 
> A new channel is fine since nothing has been registered. Can we please get this channel registered to the [Chain Registry](https://github.com/cosmos/chain-registry/tree/master/_IBC)?

@JeremyParish69 registered 😊

**Pull request merged**